src/project-manager.ts

```diff
@@ -359,7 +354,7 @@ export class ProjectManager implements Disposable {
 				);
 			})
 			// Use same scheme, slashes, host for referenced URI as input file
-			.map(filePath => url.format({ ...parts, pathname: filePath.split(/[\\\/]/).map(encodeURIComponent).join('/'), search: undefined, hash: undefined }))
+			.map(filePath => util.path2uri('', filePath))
```

Comment:
That would work, but it would be less efficient here because we already have the URI parsed, so we would parse it twice. What we could do is accept `string | url.Url`